app/comments/get.controller.js

```diff
@@ -0,0 +1,28 @@
+const { logger } = require('../../common/logging/logger')
+const { getSentencePlanComments } = require('../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../common/utils/getCommentText')
+
+const getComments = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planId }, session: { 'x-auth-token': token } },
+  res
+) => {
+  const renderDetails = renderInfo || {}
+
+  renderDetails.nexturl = path.substring(0, path.lastIndexOf('/'))
+  renderDetails.backurl = `${path.substring(0, path.lastIndexOf('/'))}/decisions`
```

Comment:
Duplication of `path.substring(0, path.lastIndexOf('/')`. 
Lots of alternatives, I'd assign a default in the deconstruct and then stick the object together with an assign or the like:
```
const getComments = async (
  { path, errors, errorSummary, body, renderInfo = {}, params: { planId }, session: { 'x-auth-token': token } },
  res
) => {
  const nexturl = path.substring(0, path.lastIndexOf('/'))
  const renderDetails = {...renderInfo, nextUrl, backurl: `${nexturl}/decisions`
```